index.html

```diff
@@ -212,7 +212,8 @@ <h1>Contact me</h1>
                     <input type="text" name="user_name" class="contact_form_name" id="name" maxlength="30" placeholder="Enter your name" required>
```

Comment:
- [ ] OPTIONAL
The _full name_ field of your _form_ is **accepting digits** as names. It will therefore be a nice implementation to add validations against your form inputs. Kindly ensure to improve upon this aspect.

Reply:
I have fixed this issue now. Now, the name field only accepts alphabets